code/game/machinery/atmoalter/gas_mine.dm

```diff
@@ -1,3 +1,5 @@
+#define KW_TO_MOLS_COEFFICIENT 0.5
```

Comment:
the units are wrong
joules to moles, or watts to moles per second

kind of a moot point, half our power units are fucked

Reply:
>makes the name even more retarded

based?